### ISSUE TITLE
An error occurred while Lint/SymbolConversion cop was inspecting xxxx

### ISSUE THREAD

**Issue by erwanlr** [rubocop-hq/rubocop#9458]
Running rubocop 1.9.0 against https://github.com/wpscanteam/OptParseValidator yields an error

## Expected behavior

Rucocop should run smoothly

## Actual behavior

```
An error occurred while Lint/SymbolConversion cop was inspecting OptParseValidator/lib/opt_parse_validator/opts/base.rb:128:6.
undefined method `str_type?' for nil:NilClass
```

## Steps to reproduce the problem

clone https://github.com/wpscanteam/OptParseValidator and run rubocop 1.9.0 against it

## RuboCop version

```
1.9.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 2.7.2 x86_64-linux)
  - rubocop-performance 1.9.2
```

**Comment by erwanlr** [comment]
Similar issue with https://github.com/wpscanteam/CMSScanner and the Style/EvalWithLocation cop

```
An error occurred while Style/EvalWithLocation cop was inspecting CMSScanner/lib/cms_scanner/finders/same_type_finder.rb:8:8.
undefined method `str_type?' for nil:NilClass
```

```
An error occurred while Style/EvalWithLocation cop was inspecting CMSScanner/lib/cms_scanner/finders/unique_finder.rb:8:8.
undefined method `str_type?' for nil:NilClass
```

**Comment by koic** [comment]
> An error occurred while Lint/SymbolConversion cop was inspecting OptParseValidator/lib/opt_parse_validator/opts/base.rb:128:6.
> undefined method `str_type?' for nil:NilClass

This issue is a dup with #9442 and resolved by #9440.

> An error occurred while Style/EvalWithLocation cop was inspecting CMSScanner/lib/cms_scanner/finders/same_type_finder.rb:8:8.
> undefined method `str_type?' for nil:NilClass

And the issue is a dup with #9433 and resolved by #9435.

Thank you.
